provision.sh

```diff
@@ -2,15 +2,23 @@
 set -e
 set -u
 set -o pipefail
+set -x
 
 # Install tools
 sudo mv /tmp/tools/* /usr/local/bin
 
+# TODO: uncomment; commented bc of this error: `NOCHANGE: partition 1 could only be grown by 1919 [fudge=2048]`
 # Grow the filesystem
-sudo growpart /dev/sda 1
-sudo xfs_growfs /
+#sudo growpart /dev/sda 1
+#sudo xfs_growfs /
 
-sudo yum install -y libvirt libvirt-devel libvirt-client git golang libvirt-daemon-kvm qemu-kvm bind-utils jq
+sudo yum install -y libvirt libvirt-devel libvirt-client git libvirt-daemon-kvm qemu-kvm bind-utils jq gcc-c++
+
+# Install golang
+curl -L https://dl.google.com/go/go1.13.8.linux-amd64.tar.gz -o go1.13.8.linux-amd64.tar.gz
+tar -xvf go1.13.8.linux-amd64.tar.gz
+sudo mv go /usr/local
+export PATH=$PATH:/usr/local/go/bin
```

Comment:
@sallyom Does default golang package in the rhel-8 is still 1.12.x ?

Reply:
yes (well as of a few months ago, I'll double check that)

Reply:
@sallyom I just checked it's not present :(